CMakeLists.txt

```diff
@@ -901,22 +901,14 @@ if (HDF5_DEFAULT_API_VERSION MATCHES "v114")
 endif ()
 
 #-----------------------------------------------------------------------------
-# Option to use 1.16.x API
-#-----------------------------------------------------------------------------
-set (H5_USE_116_API_DEFAULT 0)
-if (HDF5_DEFAULT_API_VERSION MATCHES "v116")
-  set (H5_USE_116_API_DEFAULT 1)
-endif ()
-
-#-----------------------------------------------------------------------------
-# Option to use 1.18.x API
+# Option to use 2.0.x API
 #-----------------------------------------------------------------------------
+set (H5_USE_2_0_API_DEFAULT 0)
```

Comment:
For consistency with other variables, should this be called `H5_USE_200_API_DEFAULT` instead?

Reply:
Requested choice to use 2_0 for variable names that do not participate in comparison functions.

Reply:
Is there a good reason for making this inconsistent though? The 0 is presumably never going to change so does 2_0 vs 200 buy us anything?